envconsul/tests/test.bats

```diff
@@ -0,0 +1,12 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Version matches" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} envconsul -version 2>&1 | head -1 | awk '{print $2}')"
+  echo $result
```

Comment:
Is this a debug `echo` that was left in accidentally? If not, maybe we should say

`echo "Version is: ${result}"`

to make it more clear that this is debug output

Reply:
Fixed.